notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>"Also, we will look at how to extend or model with an additional serving function that enables us to control the model's behavior by providing some preprocessing and postprocessing logic." &gt;&gt; "Also, we will look at how to extend our model with an additional serving function. Additional serving functions allow you to provide preprocessing and/or postprocessing logic to a model's prediction." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Sounds good. Thanks!</p>